### ISSUE TITLE
feature request: keep database unlock for a few sec to clip username, then password

### ISSUE THREAD

**Issue by lbesnard** [samedamci/rofi-keepassxc#3]
At the moment, when clipping the username, the database is closed, and one has to re-enter the keepass password to clip the password of an entry. It would be great if the database could stay unlocked by calling the program with --time-out for example.

**Comment by wzykubek** [comment]
Yes, I know but I don't have any idea how to do that in secure way. Unfortunately keepassxc-cli doesn't support this feature...

**Comment by iFreilicht** [comment]
There's another project called [keepmenu](https://github.com/firecat53/keepmenu/blob/main/keepmenu) that offers this functionality. What they bascially do is have the initial script start a daemon, which then opens the keepass file and rofi. When exiting, the daemon keeps running, thus the keepass file is staying opened. A subsequent call to the initial script just sends a message to the daemon to start rofi again.
